docs/content/Deployment/Production-Checklist.md

```diff
@@ -39,6 +39,14 @@ password, please set it via the `REDIS_PASSWORD` environment variable. Set the
 connections. Ensure your Redis cluster allows at least 15 concurrent
 connections.
 
+### Redis Sentinel
+
+Redis Sentinel support via ioredis was recently added. In order to activate
```

Comment:
Could we add a link to the ioredis repo here like the above suggestion? Just so people who are unaware of it can quickly brush up on it 😄

Reply:
I would also suggest a few ideas:
* Docs are usually considered a long-lasting and time-insensitive medium (quite the opposite of a blog post or a changelog entry) meaning that it's often unclear what "recently" in docs really means 😄. Maybe we can skip it here?
* Providing links to Redis Sentinel and ioredis is a great idea but I'd also suggest to add a few words about the value of using them. People who are already familiar with this software don't need this but people who'd like to have a HA Redis but just don't know it's called Sentinel might benefit from such clarification 😄

Reply:
Yup